### ISSUE TITLE
Allow item/fluid interfaces to be filtered using variable cards

### ISSUE THREAD

**Issue by LiamJTolentino** [CyclopsMC/IntegratedTunnels#211]
#### Issue type:

- :heavy_plus_sign: Feature request 

____

#### Short description:

In other storage network mods, like Refined Storage or Applied Energistics, they have a way of filtering what items can be stored in certain external inventories that are connected to the network making it easy to sort items. However, in Integrated Tunnels only the importers and exporters can take in variable cards to filter what can go through while the interfaces simply store whatever is added into the network. This becomes a problem when you want only a certain type of fluid to be stored in a certain tank connected to the network, but when it goes empty, any fluid that gets imported to the network can get stored in there and then the fluid you want stored will have nowhere to go.

This feature could also open up a huge world of possibilities, like if you want to sort out your storage room according to mod type, you could have each item interface with a predicate card that allows items of a particular mod type to be stored there, or have a fluid storage room with tanks for each fluid in the network so you can visually see what fluids you're using and stuff.

Tl;dr, Item and fluid interfaces should have filtering options for what can be stored in the connected inventory much like how other storage network mods can blacklist/whitelist items that can be stored in an inventory.

**Comment by rubensworks** [comment]
Thanks for the suggestion!

**Comment by rubensworks** [comment]
Sounds similar to #98, so I'll merge this with that issue.
